client/src/ui/organisms/placement/index.tsx

```diff
@@ -350,19 +355,21 @@ function RenderSideOrTopBanner({
         </a>
       </p>
 
-      <a
-        className="no-pong"
-        data-glean={
-          "pong: " + (user?.isSubscriber ? "pong->settings" : "pong->plus")
-        }
-        href={
-          user?.isSubscriber
-            ? "/en-US/plus/settings?ref=nope"
-            : "/en-US/plus?ref=nope#subscribe"
-        }
-      >
-        Don't want to see ads?
-      </a>
+      {showNoAds && (
+        <a
+          className="no-pong"
```

Comment:
Nit: We seem to have this in three locations, can we extract and reuse it?

Reply:
Yes, But that was true for the existing code. I wanted to keep the change minimal.